File: zar.c
#include <stdio.h>
#include <stdlib.h>

int main() {
	int a = rand() % 6 + 1;
	int b = rand() % 6 + 1;
	
	printf("Dau cu zaru %d %d", a, b);
	
	if(a == 6 && b == 5) {
		printf(" n-am nevoie de servici");
	} else if(a == 6 && b == 6) {
		printf(" ca mine nu se mai naste");
	}

	printf("\n");

	return 0;
}

File: zar.py
import random

def main():
	x = random.randint(1,6)
	y = random.randint(1, 6)
	
	if x + y == 12:
		print("Dau cu zaru 6 6 ca mine nu se mai naste")
	elif x + y == 11:
		print("Dau cu zaru 6 5 n-am nevoie de servici")
	else:
		print(f"Dau cu zaru {x} {y}")
main()
